charts/kyverno/values.yaml

```diff
@@ -1209,7 +1209,18 @@ backgroundController:
 
     clusterRole:
       # -- Extra resource permissions to add in the cluster role
-      extraResources: []
+      extraResources: 
+
+       - apiGroups:
+            - rbac.authorization.k8s.io
+          resources:
+            - rolebindings
+            - roles
+          verbs:
+            - create
+            - update
+            - patch
+            - delete
```

Comment:
Why add these permissions by default?

The issue is in https://github.com/kyverno/kyverno/pull/11463/checks?check_run_id=31984424855.

Reply:
I apologize for the oversight in PR #11463 regarding the default permissions and the missing commit sign-offs. The goal was to streamline access, but I realize that including additional permissions by default wasn’t ideal and led to issues in the checks. I also missed the commit sign-off, which didn’t adhere to the project’s guidelines.

Thank you for bringing these to my attention. I’ll correct both the permissions and commit sign-offs promptly and be more diligent about these standards in the future.